src/index.scss

```diff
@@ -0,0 +1,129 @@
+@import "variables";
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  background-color: $white-color;
+  border: 1px solid $darker-white;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  padding: 16px;
+
+  &__img {
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 16px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-color;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+    margin-top: 4px;
+  }
+
+  .stars {
+    display: flex;
+    margin-top: 16px;
+
+    &__item {
+      width: 16px;
+      height: 16px;
+      background-image: url(./images/star.svg);
+      background-repeat: no-repeat;
+      background-position: 50%;
+    }
+
+    & > :last-child {
+      margin-right: 0;
+    }
+
+    &--1 > :nth-child(-n+1),
+    &--2 > :nth-child(-n+2),
+    &--3 > :nth-child(-n+3),
+    &--4 > :nth-child(-n+4),
+    &--5 > :nth-child(-n+5) {
+      background-image: url(./images/star-active.svg);
+    }
+
+    &__item:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &__reviews {
+      display: flex;
+      font-size: 10px;
+      font-style: normal;
+      margin-left: auto;
+      font-weight: 400;
+      color: $dark-color;
+    }
+  }
+
+  &__price-container {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    font-size: 12px;
+    font-style: normal;
```

Comment:
This is default behaviour, you can check it by removing this property in devtools. The same for `stars__review`.